src/lib/icons/icons.js

```diff
@@ -78,7 +78,6 @@ const FILE_LOADING_ICONS = {
     FILE_BOX_NOTE,
     FILE_CODE,
     FILE_DEFAULT,
-    FILE_DICOM,
```

Comment:
We're missing a DICOM-specific icon in the new set, but `.dcm` files were already using the `FILE_IMAGE` icon. I'll update if I hear otherwise from design.